avocado/core/settings.py

```diff
@@ -250,6 +267,8 @@ def get_value(self, section, key, key_type=str, default=no_default,
         """
         try:
             val = self.config.get(section, key)
+        except ConfigParser.NoSectionError:
+            return self._handle_no_section(section, default)
```

Comment:
Well do you think it's worth distinguishing between section vs. value? I mean the difference is:

```
SettingsError: Section 'foo' doesn't exist in configuration
```

vs.

```
SettingsError: Value 'bar' not found in section 'foo'
```

which is both true and should serve the purpose (you know me, less code == less bugs).

Anyway it works well so just let me know whether you believe it's worth adding...